x/consensus/types/codec.go

```diff
@@ -27,10 +27,7 @@ func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	legacy.RegisterAminoMsg(cdc, &MsgUpdateParams{}, "cosmos-sdk/x/consensus/MsgUpdateParams")
 }
 
-var (
-	amino     = codec.NewLegacyAmino()
-	ModuleCdc = codec.NewAminoCodec(amino)
-)
+var amino = codec.NewLegacyAmino()
```

Comment:
Shouldn't this one be public now?

Reply:
why was moduleCdc public? its only used internally.

Reply:
🤷🏾‍♂️ it is just that someone cab rely on it now.

Reply:
Sure, but then I'd argue that we should break them consistently.
Some Amino variable la that have been put public in this PR should be put back private.

Reply:
true, ill do it in a follow up pr